static/js/topic_page/line_tile.tsx

```diff
@@ -56,22 +56,18 @@ export function LineTile(props: LineTilePropType): JSX.Element {
     if (chartData) {
       drawChart(chartData);
     }
-  }, [chartData]);
```

Comment:
merge error? should keep the reference to chartData

Reply:
I intended to fix some deps lint warnings. Completed them now.